mantidproject/mantid
# Remove qt3 from ImportASCIIDialog

Q3Table is replaced with QTableWidget in the dialog.

**To test:**
- Import an ASCII file into a Table. (File->Import ASCII...)
- Check that if the data format is selected correctly the table in the dialog displays the data.
- Try different options

Fixes #15475.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
